.prettierignore

```diff
@@ -0,0 +1,7 @@
+.gitignore
```

Comment:
I would just use --ignore-path .gitignore 

prettier should not format `.lock` `.nix` `.sh` or `.toml` files  why are they included here?

Reply:
Pre-commit passes each file which it lints to the command, enabling it to trivially check a lot of files in parallel. This is controlled by settings in `.pre-commit-config.yaml`. I just copied the setting from the [upstream configuration](https://github.com/pre-commit/mirrors-prettier/blob/6e24ff01ab7d5e64045599b81c4cd34d6a6a7159/.pre-commit-hooks.yaml), but it seems that isn't quite the same as what we're linting.

I can change the configuration to match what we are actually linting, but for that I need to know which files to pass to `yarn lint`. I see `*.md` and `*.yaml` mentioned in `package.json`; which other files should I include?

Reply:
I've unwrapped the configuration into running `prettier` on `*.md` and `*.yaml`, but from `node_modules/.bin/eslint --debug` it looks like that's not actually validating any files. Should we remove it?